### PR TITLE
Update vault script

### PR DESCRIPTION
**Pull request by lousyd**
##### SUMMARY
Removed reference to `vault-keyring-client.py`, since it no longer exists. Two times that file is mentioned it's just as an example, so I left those. But one of the mentions was to a specific file in a directory, and that's the line I deleted.

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

### CODE REVIEW

**Comment by sivel** [comment]
fwiw, that file now lives at https://github.com/ansible-community/contrib-scripts/blob/main/vault/vault-keyring-client.py

**Comment by bcoca** [comment]
it was moved to the community general collection

File: docs/docsite/rst/user_guide/vault.rst [hunk 1]
```diff
@@ -113,8 +113,6 @@ Ansible executes the client script with a ``--vault-id`` option so the script kn
 
     contrib/vault/vault-keyring-client.py --vault-id dev
 
-For an example of a client script that loads passwords from the system keyring, see :file:`contrib/vault/vault-keyring-client.py`.
```
Comment (Akasurde) [inline docs/docsite/rst/user_guide/vault.rst:116]: How about linking it to the new home of vault script - https://github.com/ansible-community/contrib-scripts/blob/main/vault/vault-keyring-client.py

**Comment by acozine** [comment]
Thanks @lousyd for pointing out the broken link! Since I see you edited this file on GitHub directly, I pushed an additional commit with the link in it - I hope you don't mind. Assuming the PR passes our CI system, I'll merge it today.

**Comment by Akasurde** [comment]
@lousyd Thanks for the contribution.

**Pull request merged**